Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -773,13 +772,11 @@ WarpXOpenPMDPlot::SetupRealProperties (openPMD::ParticleSpecies& currSpecies,
         auto ii = m_NumAoSRealAttributes + idx; // jump over AoS names
         if (write_real_comp[ii]) {
             // handle scalar and non-scalar records by name
-            std::string record_name, component_name;
-            std::tie(record_name, component_name) = detail::name2openPMD(real_comp_names[ii]);
+            const auto [record_name, component_name] = detail::name2openPMD(real_comp_names[ii]);
             auto currRecord = currSpecies[record_name];
 
             // meta data for ED-PIC extension
-            bool newRecord = false;
-            std::tie(std::ignore, newRecord) = addedRecords.insert(record_name);
+            const auto [std::ignore, newRecord] = addedRecords.insert(record_name);
```

Comment:
Yep, not fair :zany_face: 
```suggestion
            [[maybe_unused]] const auto [_, newRecord] = addedRecords.insert(record_name);
```